core/src/main/java/psiprobe/beans/ContainerWrapperBean.java

```diff
@@ -127,7 +127,7 @@ public void setWrapper(Wrapper wrapper) {
       }
     } catch (ThreadDeath e) {
```

Comment:
Perfect!  I was looking at that yesterday and wasn't quite sure as I've never saw thread death before.


Sent from Outlook Mobile






On Sun, Feb 7, 2016 at 9:31 AM -0800, "Mark Lewis" <notifications@github.com> wrote:





> @@ -127,7 +127,7 @@ public void setWrapper(Wrapper wrapper) {
>        }
>      } catch (ThreadDeath e) {

`catch (ThreadDeath)` is only necessary if you're going to use `catch (Throwable)`. ThreadDeath extends Error, so it won't be caught by `catch (Exception)`. You can remove `catch (ThreadDeath)` entirely, and it'll be cleaner.

There are many instances of this throughout this PR. I won't note them all because that's just noisy.

---
Reply to this email directly or view it on GitHub:
https://github.com/psi-probe/psi-probe/pull/655/files#r52120883